analyses/mutational-signatures/08-explore_hypermutators.Rmd

```diff
@@ -14,18 +14,24 @@ editor_options:
 
 
 ```{r setup}
+library(tidyverse)
+library(pheatmap)
+library(ggplot2)
```

Comment:
If we load `tidyverse`, no need to load `ggplot2`
```suggestion
```

Reply:
updated